pom.xml

```diff
@@ -421,9 +443,11 @@
                 </exclusions>
             </dependency>
             <dependency>
+                <!-- gt-imageio-ext-gdal is 'works with' so it must stay in provided scope -->
                 <groupId>org.geotools</groupId>
                 <artifactId>gt-imageio-ext-gdal</artifactId>
                 <version>${gt.version}</version>
+                <scope>provided</scope>
```

Comment:
@AndrewAnnex Actually, this dep is approved for compile time - it's just the it.geosolutions.imageio-ext stuff we can't use...